### ISSUE TITLE
Regression: checkbox widget not working since v1.0.0

### ISSUE THREAD

**Issue by firedaemon-cto** [BorisMoore/jsviews#419]
The checkbox widget doesn't work anymore - when clicking on the checkbox it just doesn't update the model (the other way around seems to work properly).

Here's a [fiddle](https://jsfiddle.net/pieceofcode/jkg4est2/), but also your [sample page](https://www.jsviews.com/#samples/tag-controls/jqui/toolbar) exhibits the same problem (clicking on the toolbar checkboxes).

This is probably correlated: An observable change wasn't reflected by v.0.9.91 while it now is in v.1.0.2. See [this fiddle](https://jsfiddle.net/pieceofcode/vb3szagd) testing an observable update.

**Comment by BorisMoore** [comment]
Thanks very much for calling this out. Yes, it is a bug in jsviews-jqueryui-widgets.js.

I'll fix it in the next update. 

For now, can you  test out adding the following line immediately after the line https://github.com/BorisMoore/jsviews.com/blob/gh-pages/download/sample-tag-controls/jsviews-jqueryui-widgets.js#L275

```
  mainElement: "input",
  linkedElement: "input", // Add this line
```

Let me know if you see any issues after making this change...

Thanks again...

**Comment by BorisMoore** [comment]
The fix is included in the new v1.0.3 release.
